src/reconcile/rec_visibility.c

```diff
@@ -273,240 +163,24 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
-    cbt = &r->update_modify_cbt;
-    __wt_modify_vector_init(session, &modifies);
     page = r->page;
-    saved_isolation = 0; /*[-Wconditional-uninitialized] */
-    first_txn_upd = tmp_upd = upd = NULL;
+    first_stable_upd = first_txn_upd = upd = NULL;
     upd_memsize = 0;
     max_ts = WT_TS_NONE;
     max_txn = WT_TXN_NONE;
-    session_flags = 0; /* [-Werror=maybe-uninitialized] */
-    las_cursor_open = las_positioned = list_prepared = list_uncommitted = false;
-    orig_val_appended = skipped_birthmark = sweep_locked = walked_past_sel_upd = false;
-
-    if (__wt_page_las_active(session, r->ref)) {
-        /* Obtain the key to iterate the lookaside */
-        WT_RET(__wt_scr_alloc(session, WT_INTPACK64_MAXSIZE, &key));
-        switch (page->type) {
-        case WT_PAGE_COL_FIX:
-        case WT_PAGE_COL_VAR:
-            p = key->mem;
-            WT_ERR(__wt_vpack_uint(&p, 0, WT_INSERT_RECNO(ins)));
-            key->size = WT_PTRDIFF(p, key->data);
-            break;
-        case WT_PAGE_ROW_LEAF:
-            if (ins == NULL) {
-                WT_WITH_BTREE(session, S2BT(session),
-                  ret = __wt_row_leaf_key(session, page, ripcip, key, false));
-                WT_ERR(ret);
-            } else {
-                key->data = WT_INSERT_KEY(ins);
-                key->size = WT_INSERT_KEY_SIZE(ins);
-            }
-            break;
-        default:
-            WT_ERR(__wt_illegal_value(session, page->type));
-        }
-
-        /* Open a lookaside cursor, position at the latest update for this key */
-        __wt_las_cursor(session, &las_cursor, &session_flags);
-        saved_isolation = session->txn.isolation;
-        session->txn.isolation = WT_ISO_READ_UNCOMMITTED;
-        las_cursor_open = true;
-        cache->las_reader = true;
-        __wt_readlock(session, &cache->las_sweepwalk_lock);
-        cache->las_reader = false;
-        sweep_locked = true;
-
-        /*
-         * Position on the latest update for this key. If no updates exist for this key, it is okay
-         * to position on an update for an adjacent key. We will check the validity of our position
-         * again when fetching the record. The updates are sorted oldest to newest in the lookaside.
-         */
-        ret = __wt_las_cursor_position(session, las_cursor, S2BT(session)->id, key, WT_TXN_MAX);
-        las_positioned = ret == 0;
-        WT_ERR_NOTFOUND_OK(ret);
-    }
+    list_prepared = list_uncommitted = false;
 
     /*
-     * If called with a WT_INSERT item, use its WT_UPDATE list (which must exist), otherwise check
-     * for an on-page row-store WT_UPDATE list (which may not exist), and any updates in lookaside.
-     * Return immediately if the item has no updates.
+     * If called with a WT_INSERT item, use its WT_UPDATE list (which must
+     * exist), otherwise check for an on-page row-store WT_UPDATE list
+     * (which may not exist). Return immediately if the item has no updates.
```

Comment:
Nit: This isn't getting formatted since the line begins with a non-alphabetical character. If you adjust it slightly and run `dist/s_all`, it should work properly.